jmix-flowui/flowui-kit/src/main/java/io/jmix/flowui/kit/component/main/ListMenu.java

```diff
@@ -347,27 +355,30 @@ protected Details createMenuBarComponent(MenuBarItem menuBarItem) {
         menuItemComponent.addClassName(JMIX_MENUBAR_ITEM_CLASS_NAME);
         menuItemComponent.addClassNames(menuBarItem.getClassNames().toArray(new String[0]));
         menuItemComponent.setOpened(menuBarItem.isOpened());
+        menuItemComponent.addThemeVariants(DetailsVariant.REVERSE);
```

Comment:
Let's make it component setting of the ListMenu component. Be default, keep at it now, so we don't unexpectedly change users' UI.

Create `ListMenuVariant` with a single constant (at least for now), implement `HasThemeVariant<ListMenuVariant>` and other xml, studio stuff. To make it simple, you don't need to set `reverse` theme name to nested details explicitly, just style them according to the parent element.